client/src/app/shared/lab-card/lab-card.component.html

```diff
@@ -1,21 +1,29 @@
 <mat-card *ngIf="lab">
-  <a class="ml-lab-card-header" title="View lab {{ lab.name}}" [routerLink]="['/editor', lab.id]">
+  <div class="ml-lab-card-header" title="View lab {{ lab.name}}" >
+    <a class="ml-lab-card-link-area" [routerLink]="['/editor', lab.id]"></a>
     <div class="ml-title-wrapper">
-      <h3>{{ lab.name | words: 5 }}</h3>
+      <a [routerLink]="['/editor', lab.id]" class="ml-lab-card-link">
+        <h3>{{ lab.name | words: 5 }}</h3>
+      </a>
       <span class="ml-user-profile-lab-card-private-badge" *ngIf="lab.is_private">Private</span>
       <div class="ml-lab-card-user" *ngIf="user && showUsername">
-        by <a class="ml-lab-card-username" title="{{ user.displayName }}'s profile" [routerLink]="['/user', user.id]">{{user.displayName}}</a>
-      </div>
-      <div class="ml-lab-card-description" *ngIf="lab.description && showDescription">
-        {{ lab.description }}
+        <a class="ml-lab-card-link-area" [routerLink]="['/editor', lab.id]"></a>
+        by
+        <img *ngIf="user.photoUrl" [src]="user.photoUrl" alt="User avatar">
```

Comment:
As we're updating this anyways, maybe we can make the `alt` a bit more specific and say `{{user.displayName}} avatar` or something along those lines.